### DISCUSSION TITLE
Sharing KeyCloak SSO session with clients placed inside iframe

### DISCUSSION THREAD

**Discussion by nmanthena18** [keycloak/keycloak#22023] [Ideas]
I have two KeyCloak clients app1, app2 in my realm related to my two web applications http://mysite/app1 and http://mysite/app2.

I want to show app2 inside iframe in app1, but when I login to app1 and redirect to app1 my application app2 does not see SSO session and redirect to KeyCloak login page

**Comment by MagiiEl** [comment]
I assume that standard flow is enabled for both clients:
is "Cookies" enabled in the login authentication flow for client2 ? Only then it will check out if there are any valid keycloak cookies (saved for the keycloak domain).

 It's a bit complicated to have this kind of set up, the problem could be different things. Maybe with more information about the network communication (what requests are sent etc.) , it helps pinpoint where the problem could be.

**Comment by nmanthena18** [comment]
Let me try by enabling cookies and I will share the updates soon.

On Thu, Jul 27, 2023 at 2:54 PM MagiiEl ***@***.***> wrote:

> I assume that standard flow is enabled for both clients:
> is "Cookies" enabled in the login authentication flow for client2 ? Only
> then it will check out if there are any valid keycloak cookies (saved for
> the keycloak domain).
>
> It's a bit complicated to have this kind of set up, the problem could be
> different things. Maybe with more information about the network
> communication (what requests are sent etc.) , it helps pinpoint where the
> problem could be.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/keycloak/keycloak/discussions/22023#discussioncomment-6563256>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGNOGKHRT2DPIWMFZC7OSBTXSIXUPANCNFSM6AAAAAA2ZXG33Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards,
Naresh Manthena,
M: 9000994443.
